docs/testbed/README.testbed.ocs_data_plan_deployment.md

```diff
@@ -0,0 +1,262 @@
+    ```
+2. Start PTF docker
+    - Launch PTF Docker through the following command. By default, do not configure the Docker network.
+    ```
+    docker run -d -it --name ptf_1 --network none sonicdev-microsoft.azurecr.io:443/docker-ptf
+    ```
+   - Check PTF container running.
+    ```
+    docker ps -a
+    CONTAINER ID   IMAGE                                                 COMMAND                  CREATED         STATUS         PORTS     NAMES
+    0723ee502496   sonicdev-microsoft.azurecr.io:443/docker-ptf          "/root/env-python3/b…"   4 seconds ago   Up 3 seconds             ptf_1
+    c0daae020353   sonicdev-microsoft.azurecr.io:443/docker-sonic-mgmt   "/bin/bash"              25 hours ago    Up 25 hours    22/tcp    sonic
+    ```
+## Configure network for PTF
+Configure a bridge network on host to link PTF to OCS. Here is the netplan method for configuring the bridged network.
+1. On **host**, use netplan to build up bridge on network card.
+    ```
+    cat /etc/netplan/01-network-br.yaml 
+    network:
+    version: 2
+    renderer: NetworkManager
+    ethernets:
+        enp0s3:
+        dhcp4: no 
+    bridges:
+        br0:
+        addresses: 
+            - 192.168.1.78/24
+            - '2001:db8:1::1/64'
+        gateway4: 192.168.1.1
+        interfaces: [enp0s3]  
+        parameters:
+            stp: true         
+            forward-delay: 3  
+    ```
+    - Apply netplan.
+    ```
+    sudo netplan apply
+    ```
+    - Check that the bridged network created on the host is successful.
+    ```
+    # brctl show
+    bridge name	bridge id		STP enabled	interfaces
+    br0		8000.00e04c68007f	yes		enp0s3
+    docker0		8000.86cd26814e67	no		veth9f77700
+    ```
+2. Build up network for PTF container.
+    Create a virtual veth pair to help the PTF achieve network communication.
+    First, configure on ***host***
+    - Create veth pair on host
+    ```
+    sudo ip link add veth0 type veth peer name veth1
+    ```
+    - Add one of the veth interfaces, veth1, to the created bridged network br0.
+    ```
+    sudo brctl addif br0 veth1
+    ```
+    - Check the bridge configuration of br0 for veth1
+    ```
+    # brctl show
+    bridge name	bridge id		STP enabled	interfaces
+    br0		8000.00e04c68007f	yes		enp0s3
+                                                            veth1
+    docker0		8000.86cd26814e67	no		veth9f77700
+    ``` 
+    - Set veth1 up
+    ```
+    sudo ip link set dev veth1 up
+    ``` 
+    - Search the process ID of the PTF Docker.
+    ```
+    docker inspect --format '{{ .State.Pid }}' ptf_1
+    187422
+    ```
+    - Configure veth0 to PTF by netns method for network namespace configuration.
+    ```
+    sudo ip link set veth0 netns 187422
+    ```
+    Then, need to login in **PTF Docker** for network configuration
+    - Login into PTF to configure router
+    ```
+    docker exec -it ptf_1 bash
+    ip addr add 192.168.1.101/24 dev veth0
```

Comment:
In inventory `ocs` and `testbed-ocs.yml`, PTF ip is 10.x.x.x. Here the mgmt IP for ptf is 192.168.1.101, they do not match.